Application/Manual/Support/Service.php

```diff
@@ -124,4 +133,86 @@ public function createTicket(IFormInterface $Form = null, $Ticket, $Attachment)
 
         return $Form;
     }
+
+    /**
+     * @param IFormInterface|null $Form
+     * @param null|array          $Request
+     *
+     * @return IFormInterface|string
+     */
+    public function createRequest(IFormInterface $Form = null, $Request = null)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Incorrect spacing between argument "$Form" and equals sign; expected 0 but found 1](https://app.codacy.com/manual/gerdchristian-kunze/SPHERE-Framework/pullRequest?prid=5646354)